src/Microsoft.CodeAnalysis.Analyzers/ReleaseTrackingAnalyzers.Help.md

```diff
@@ -10,109 +12,114 @@ Release tracking analyzer enables third party analyzer packages to define analyz
    3. Enabled by default status.
 
 This analyzer expects that the third party analyzer project has two additional files, namely `AnalyzerReleases.Unshipped.md` and `AnalyzerReleases.Shipped.md`, to track this release specific metadata.
+
 1. `AnalyzerReleases.Unshipped.md`: Additional file for the upcoming or unshipped analyzer release. This file will start empty at the beginning of each release. While working on the upcoming release, it will track additions/removals/changes to analyzer rules in the repo.
 2. `AnalyzerReleases.Shipped.md`: Additional file for shipped analyzer releases. This file only tracks analyzer rules for shipped releases. It cannot be changed while work is in progress for upcoming release. When a new analyzer package is released, a new release with shipped package version should be created in the shipped file, and all the entries from the unshipped file should be mvoed under the new shipped release.
 
 Note that each analyzer project that contributes an assembly to the analyzer NuGet package would require these additional files.
 
 Release tracking analyzer provides the diagnostics and code fixes to enable analyzer authors to maintain the shipped and unshipped files. Analyzer author only requires to perform the following two manual tasks:
+
 1. Create these two files and pass mark them as additional files for their analyzer project (steps explained in next section), and
 2. Move all the entries from unshipped file to the shipped file after each release.
 
 ## Example
+
 Consider the following example:
 
 1. `AnalyzerReleases.Shipped.md`:
 
-```
-## Release 1.0
+   ```md
```

Comment:
Not sure about these leading spaces.

Reply:
@Evangelink The leading spaces makes the content to appear correctly under the list item (i.e. there is a **visual** change here). The absence of the leading spaces also violates some rule, basically because the code snippet becomes outside the list item, which means the whole list is ended. Hence, the other list item that starts with `2.` is considered as a new list (instead of becoming a new list *item* under the same list). The linter triggers an error when a new list starts with number other than `1`.

Reply:
The HTML before the leading spaces looks like this:

![image](https://user-images.githubusercontent.com/31348972/94803220-497b1b00-03e9-11eb-90aa-68d36f622fe1.png)

Note there are 2 `<ol>` tags, which the second one starts with 2.

Reply:
Please revert the changes here. This md file is an additional file used by the analyzer, and I am not sure the analyzer that parses it handles these spaces. Also the code fix that produces this md will not add these spaces, so user would try hand editing, which is undesirable.

Reply:
@mavasani This is not going to render with spaces, so the result for users is unchanged.